src/services/StreamService.ts

```diff
@@ -0,0 +1,65 @@
+import {DynamoDBRecord} from "aws-lambda";
+import {DynamoDB} from "aws-sdk";
+import {cloneDeep} from "lodash";
+import {Utils} from "../utils/Utils";
+import {ITechRecordWrapper, Plates} from "../utils";
+
+/**
+ * Service class for interpreting and formatting
+ * incoming DynamoDB streams
+ */
+class StreamService {
+
+    /**
+     * Extract MODIFY events from the DynamoDB Stream, convert them
+     * to a JS object and expand the tech records into multiple ones for each plate
+     * Example:
+     * Convert
+     * tech-record
+     *  ├── new-plate-1
+     *  ├── new-plate-2
+     *  into
+     *  tech-record
+     *  └── new-plate-1
+     *  tech-record
+     *  └── new-plate-2
+     * @param event
+     */
+    public static getTechRecordsStream(event: any) {
+        // Create from a tech record with multiple plates, multiple tech records with one plate each
+        console.log("Filtering only MODIFY events");
```

Comment:
Is this needed?

Reply:
If you are referring to why are we allowing only "modify" events, it's because plate generation happens only on PUT. We don't generate plates on POST. 
If you are referring to the logs, I will remove most of the logs after we test the entire flow

Reply:
I was talking about the console log which is removed now :)